server/src/controllers/buttonController.js

```diff
@@ -10,6 +12,16 @@ const buttonDispatcher = (req, res) => {
   //dispatch to the responsible function based on the callback_id
   if (parsedPayload.callback_id === 'clickSaveJobs') {
```

Comment:
might make sense to refactor to a switch-case statement

generally when you see hella else ifs, you can consider switch case